vms/platformvm/genesis/genesis.go

```diff
@@ -39,5 +43,7 @@ func Parse(genesisBytes []byte) (*Genesis, error) {
 			return nil, err
 		}
 	}
+
+	gen.GenesisID = hashing.ComputeHash256Array(genesisBytes)
```

Comment:
this is what we used to do in state package to derive genesisID. Just moved here